myia/compile/backends/relay.py

```diff
@@ -403,6 +411,56 @@ def relay_take_grad_inp(c, _nb_indices, _indices, _values):
     return relay.concatenate(outputs, 0)
 
 
+def relay_random_initialize(c, ref_seed):
+    """Create a random state for Philox2x32 RNG.
+
+    State is a couple (key, counter).
+    key is given seed, or a default value if seed is None.
+    counter starts with 0 and is incremented after each generation batch.
+    """
+    assert ref_seed.is_constant(type(None)) or ref_seed.is_constant(int)
+    seed = ref_seed.value
+    if seed is None:
+        seed = 12345678
```

Comment:
I don't think there should be a default seed, especially not in the backend implementation.

Reply:
Ok!